acceptance/bin/sort_lines.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+"""
+Helper to sort lines in text file. Similar to 'sort' but no dependence on locale or presence of 'sort' in PATH.
```

Comment:
Curious: was either sort not in the PATH or did you have locale issues?

Reply:
Locale issues on windows CI.